distribution/docs/src/main/jdocs/content/_tables/org.codice.alliance.nsili.orb.impl.corbaorb-table-contents.adoc

```diff
@@ -0,0 +1,30 @@
+:type: documentation
+:status: published
+
+.[[org.codice.alliance.nsili.orb.impl.corbaorb]]NSILI Corba ORB
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Corba Listen Port
+|corbaPort
+|Integer
+|Corba Listen Port.
+|${org.codice.alliance.corba_default_port}
+|true
+
+| CORBA Timeout
+| corbaTimeout
+| Integer
+| The timeout (seconds) associated with CORBA reads.
+| 60
+| true
+
```

Comment:
Are these spaces before the text intentional? I've noticed some of the table entries have them and others don't